### PR TITLE
convert VehicleReport to Twist msgs

### PR DESCRIPTION
**Pull request by RyuYamamoto**
## Description
convert VehicleReport message to TwistStampd / TwistWithCovarianceStamped message.

## Pre-Review Checklist for the PR Author

**PR Author should check the checkboxes below when creating the PR.**

- [ ] Read [pull request guidelines][pull-request-guidelines]
- [ ] Code follows [coding guidelines][coding-guidelines]
- [ ] Assign PR to reviewer

## Checklist for the PR Reviewer

**Reviewers should check the checkboxes below before approval.**

- [ ] Commits are properly organized and messages are according to the guideline
- [ ] Code follows [coding guidelines][coding-guidelines]
- [ ] (Optional) Unit tests have been written for new behavior
- [ ] PR title describes the changes

## Post-Review Checklist for the PR Author

**PR Author should check the checkboxes below before merging.**

- [ ] All open points are addressed and tracked via issues or tickets
- [ ] Write [release notes][release-notes]

## CI Checks

- **Build and test for PR / build-and-test-pr**: Required to pass before the merge.
- **Build and test for PR / clang-tidy-pr**: NOT required to pass before the merge. It is up to the reviewer(s). Found false positives? See the [guidelines][clang-tidy-guidelines].
- **Check spelling**: NOT required to pass before the merge. It is up to the reviewer(s). See [here][spell-check-dict] if you want to add some words to the spell check dictionary.

[clang-tidy-guidelines]: https://tier4.github.io/autoware.proj/tree/main/developer_guide/ClangTidyGuideline/
[coding-guidelines]: https://tier4.atlassian.net/wiki/spaces/AIP/pages/1194394777/T4
[pull-request-guidelines]: https://tier4.github.io/autoware.proj/tree/main/developer_guide/PullRequestGuideline/
[release-notes]: https://tier4.atlassian.net/wiki/spaces/AIP/pages/563774416
[spell-check-dict]: https://github.com/tier4/autoware-spell-check-dict#how-to-contribute

### CODE REVIEW

File: localization/twist_estimator/vehicle_velocity_converter/src/vehicle_velocity_converter.cpp [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  const autoware_auto_vehicle_msgs::msg::VelocityReport::SharedPtr msg)
+{
+  if (msg->header.frame_id != frame_id_) {
+    RCLCPP_WARN(get_logger(), "frame_id is not base_link.");
+  }
+  // set twist stamp msg from vehicle report msg
+  geometry_msgs::msg::TwistStamped twist_msg;
+  twist_msg.header = msg->header;
+  twist_msg.twist.linear.x = msg->longitudinal_velocity;
+  twist_msg.twist.linear.y = msg->lateral_velocity;
```
Comment (taichiH) [inline localization/twist_estimator/vehicle_velocity_converter/src/vehicle_velocity_converter.cpp:46]: @RyuYamamoto 
Please add `heading_rate` value to `angular.z`.

Comment (RyuYamamoto) [reply localization/twist_estimator/vehicle_velocity_converter/src/vehicle_velocity_converter.cpp:46]: thank you for review. I forgot to set value...
fixed 49fa69a08348dec1788b1356a2dbdea62666d861 .

File: localization/twist_estimator/vehicle_velocity_converter/config/vehicle_velocity_converter.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+        0.0,
+        0.0,
+        0.0,
+        0.0,
+        0.0,
+        0.0,
+        0.0,
+        0.0,
+        0.0,
+      ]
```
Comment (YamatoAndo) [inline localization/twist_estimator/vehicle_velocity_converter/config/vehicle_velocity_converter.yaml:42]: Please set default value with reference to https://github.com/tier4/autoware.iv/blob/main/localization/twist_estimator/gyro_odometer/src/gyro_odometer_core.cpp#L69-L74

Comment (RyuYamamoto) [reply localization/twist_estimator/vehicle_velocity_converter/config/vehicle_velocity_converter.yaml:42]: fixed dcbb9f694082ca99d36ccdc4e510bac637c4f795.

**Review by YamatoAndo — APPROVED** [review]

**Pull request merged**